File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/03.Conditional-Statements-Advanced/03-additional/05.vacation.js
function vacation(input) {

    const budget = Number(input[0]);
    const season = input[1];

    let price = 0;
    let location = "";
    let place = "";

    if (budget <= 1000) {
        place = "Camp";
        switch (season) {
            case "Summer": location = "Alaska";
                price = 0.65 * budget;
                break;
            case "Winter": location = "Morocco";
                price = 0.45 * budget;
                break;
        }
    } else if (budget <= 3000) {
        place = "Hut";
        switch (season) {
            case "Summer": location = "Alaska";
                price = 0.8 * budget;
                break;
            case "Winter": location = "Morocco";
                price = 0.6 * budget;
                break;
        }
    } else if (budget > 3000) {
        place = "Hotel";
        switch (season) {
            case "Summer": location = "Alaska";
                price = 0.9 * budget;
                break;
            case "Winter": location = "Morocco";
                price = 0.9 * budget;
                break;
        }
    }

    console.log(`${location} - ${place} - ${price.toFixed(2)}`);
}
vacation(["5000", "Winter"]);

File: Python_Basics/Homeworks-And-Labs/03.conditional_statements_advanced/03.additional/05.vacation.py
budget = float(input())
season = input()

price = 0
location = ""
place = ""

if budget <= 1000:
    place = "Camp"
    if season == 'Summer':
        location = "Alaska"
        price = 0.65 * budget
    elif season == 'Winter':
        location = "Morocco"
        price = 0.45 * budget
elif budget <= 3000:
    place = "Hut"
    if season == 'Summer':
        location = "Alaska"
        price = 0.8 * budget
    elif season == 'Winter':
        location = "Morocco"
        price = 0.6 * budget
elif budget > 3000:
    place = "Hotel"
    if season == 'Summer':
        location = "Alaska"
        price = 0.9 * budget
    elif season == 'Winter':
        location = "Morocco"
        price = 0.9 * budget

print(f'{location} - {place} - {price:.2f}')
